docs/quickstart/php.md

```diff
@@ -4,4 +4,203 @@ headline: PHP Quick Start
 layout: docs
 sidenav: doc-side-quickstart-nav.html
 type: markdown
```

Comment:
sorry i missed this out in the earlier review, could you please change this to be:
```
---
bodyclass: docs
layout: docs
headline: PHP Quickstart
sidenav: doc-side-quickstart-nav.html
type: markdown
---
```